snuba/clickhouse.py

```diff
@@ -375,6 +377,32 @@ def for_schema(self):
         return ', '.join(column.for_schema() for column in self.columns)
 
 
+class TableSchema(object):
+    """
+    Represents the full set of columns in a clickhouse table, this only contains
+    basic metadata for now. The code to generate the schema comes in a followup PR.
+    """
+    DATABASE = None
+
+    TEST_TABE_PREFIX = "test_"
+    LOCAL_TABLE_NAME = None
```

Comment:
Simpler approach.
Prod and dev tables do not have different names. If someone really really wants to do that they can override local_table, dist_table and query_table.
Otherwise the event class only specifies the name of the local and distributed table. The parent class decides what to return depending whether we are in local mode, distributed mode or test mode.